AUTHORS

```diff
@@ -36,7 +36,9 @@ Mindy Preston <mindy.preston@docker.com>
 Natanael Copa <natanael.copa@docker.com>
 Nathan Dautenhahn <ndd@seas.upenn.edu>
 Nathan LeClaire <nathan.leclaire@docker.com>
+Nico Di Rocco <dirocco.nico@gmail.com>
 Phil Estes <estesp@linux.vnet.ibm.com>
+radu-matei <matei.radu94@gmail.com>
```

Comment:
If you want to have this nicer looking; name looks to be `Radu Matei`; https://github.com/radu-matei